README.md
[![Logotipo da Rede Cultura Viva](doc/img/logo-cultura-viva.png)](http://culturaviva.gov.br/)

[![Bugs Registrados](https://badge.waffle.io/culturagovbr/mapasculturais-culturaviva.svg?label=bug&title=Bugs)](http://waffle.io/culturagovbr/mapasculturais-culturaviva)
[![Melhorias Solicitadas](https://badge.waffle.io/culturagovbr/mapasculturais-culturaviva.svg?label=enhancement&title=Melhorias)](http://waffle.io/culturagovbr/mapasculturais-culturaviva)
[![Tarefas Finalizadas](https://badge.waffle.io/culturagovbr/mapasculturais-culturaviva.svg?label=ready&title=Finalizadas)](http://waffle.io/culturagovbr/mapasculturais-culturaviva)


Tema de [Mapas Culturais](https://github.com/hacklabr/mapasculturais) para a **Rede Cultura Viva**.


## O Programa

O Programa Cultura Viva, desde sua implementação em 2004, alcançou grande dimensão nacional, e atualmente é uma referência para políticas culturais em vários países da América Latina. Sua política foi desenhada para valorizar a cultura realizada na base da sociedade brasileira, a articulação em rede e a gestão compartilhada, com base nos princípios da autonomia, protagonismo e empoderamento da sociedade civil.

A Política de Cultura Viva contempla iniciativas ligadas à cultura de base comunitária, Indígenas, Quilombolas, de Matriz Africana, economia solidária, produção cultural urbana e periférica, cultura digital, cultura popular, com ampla incidência no segmento da juventude, abrangendo todos os tipos de linguagem artística e cultural como música, artes cênicas, cinema, circo, literatura, entre outras.

Tem o objetivo de promover a articulação destas iniciativas em rede, contribuindo para a inclusão social, o combate ao preconceito e a todas as formas de discriminação e intolerância, o reconhecimento e a valorização da diversidade cultural brasileira e o pleno exercício dos direitos culturais.

Desde 23 de julho de 2014, com a sanção da Lei Cultura Viva (Lei nº 13.018/2014), o Programa Cultura Viva e os Pontos e Pontões de Cultura tornaram-se Política de Estado. Além de garantir a continuidade do Programa, a Lei simplifica e desburocratiza os processos de prestação de contas e o repasse de recursos para as organizações da sociedade civil.

Atualmente está presente nos 27 estados brasileiros e em cerca de mil municípios, é um dos programas com mais capilaridade e visibilidade do Ministério da Cidadania, atualmente com 4.502 Pontos fomentados. Até 2020 deverão ser fomentados mais 10.498 Pontos de Cultura para atingir a meta de 15 mil Pontos prevista no Plano Nacional de Cultura.


## Desenvolvedor: Primeiros Passos

Siga o [Guia de Instalação e Configuração](doc/guia-instalacao.md) para ter seu ambiente de trabalho funcionando sem
dores de cabeça.

### Documentação

- [Guia de Instalação e Configuração](doc/guia-instalacao.md)
- [Log de Auditoria](doc/modelo-dados.md#log-de-auditoria)
- [Modelo de Dados](doc/modelo-dados.md#modelo-fisico)
- [Dicionário de Dados](doc/modelo-dados.md#dicionario-de-dados)



## Links Úteis

* TODO


## Contribuindo

Contribuições são bem vindas.

* Faça um **pull-request** contendo correção de erros, adição de novas funcionalidades ou melhoria na documentação.
* [Proponha](https://waffle.io/culturagovbr/mapasculturais-culturaviva) novas funcionalidades e melhorias.
* [Reporte erros](https://waffle.io/culturagovbr/mapasculturais-culturaviva) encontrados no sistema.




CHANGELOG.md
# Changelog

## [1.0.0] - 30-07-2018
- Bug de exibição de localização no mapa, a partir de CEP único de município; ([#241](https://github.com/culturagovbr/mapasculturais-culturaviva/issues/241));
- Faz com que Os cadastros de 'pontãos' sejam obrigatoriamente entidade ([#222](https://github.com/culturagovbr/mapasculturais-culturaviva/issues/222));
- Passa a buscar a geolocalização via Google Maps API, a partir de inserção de Bairro e Rua, além do CEP;
- Corrige inscrições que foram inseridas incorretamente;
- Correção texto L16;
- Remoção termo autodeclaração;
- Atualização da documentação, incluido configuração do config.php para detectar o dominio do RCV;

views/rede/faq.php
<section id="page-faq">
    <article class="slide-home">
        <img src="<?php $this->asset('img/cultura-viva-share.png') ?>" />
        <ul>
            <li><a href="#somos_todos"> Somos todos Pontos de Cultura <span class="icon icon-plus-circled"></span></a></li>
            <li><a href="#cultura_viva">O que é Cultura Viva<span class="icon icon-plus-circled"></span></a></li>
            <li><a href="#ponto_e_pontao">O que é Ponto e Pontão de Cultura?<span class="icon icon-plus-circled"></span></a></li>
            <li><a href="#por_que_ponto_e_postao">Por que ser Ponto ou Pontão de Cultura?<span class="icon icon-plus-circled"></span></a></li>
            <li><a href="#o_que_ganho">O que eu ganho com isso?<span class="icon icon-plus-circled"></span></a></li>
            <li><a href="#autodeclarar">Como me Autodeclarar Ponto ou Pontão de Cultura?<span class="icon icon-plus-circled"></span></a></li>
        </ul>
    </article>

    <article class="content_faq">
        <div class="row">
            <h3 id="somos_todos">Somos todos Pontos de Cultura</h3>
            <p>O Programa Cultura Viva é uma política cultural voltada para o reconhecimento e apoio à  atividades e processos culturais já desenvolvidas, que estimula a participação social, a colaboração e a gestão compartilhada de políticas públicas no campo da cultura. Fomentar essas práticas é também aprofundar a democracia cultural e, por que não, a cultura democrática.</p>
            <p>A Rede Cultura Viva é a plataforma do Cadastro Nacional de Pontos e Pontões de Cultura, instituído pela Lei Cultura Viva. É a ferramenta que permite a autodeclaração por parte das entidades e coletivos culturais, e o reconhecimento por parte do Estado, estabelecendo uma relação direta entre os Pontos de Cultura, o Ministério da Cidadania, e os entes federados parceiros. A rede social dos fazedores e fazedoras de cultura do Brasil. </p>
            <p>Para quem já é Ponto ou Pontão de Cultura, o Cadastro é uma forma de manter os dados atualizados, acompanhar a execução de plano de trabalho, enviar informação sobre atividades desenvolvidas, além de se manter informado a respeito de encontros, editais, apoios e articulações. </p>
            <p>Para quem quer ser Ponto ou Pontão de Cultura, é só chegar! Insira seus dados no Cadastro e, a partir de uma seleção simplificada e da adesão à Política Nacional de Cultura Viva, a entidade e coletivo cultural poderão ser reconhecidos como Ponto ou Pontão de Cultura, passando a integrar a rede da Política Nacional de Cultura Viva.</p>
            <p>Curtiu? Então seja mais um ponto nesta teia da diversidade cultural brasileira!</p>
        </div>
        <div class="row">
            <h3 id="cultura_viva">O que é Cultura Viva</h3>
            <p>O Programa Cultura Viva, desde sua implementação em 2004, alcançou grande dimensão nacional, e atualmente é uma referência para políticas culturais em vários países da América Latina. </p>
            <p>Presente nos 27 estados brasileiros e em cerca de mil municípios, é um dos programas com mais capilaridade e visibilidade do Ministério da Cidadania, atualmente com 4.502 Pontos fomentados. Até 2020 deverão ser fomentados mais 10.498 Pontos de Cultura para atingir a meta de 15 mil Pontos prevista no Plano Nacional de Cultura.</p>
            <p>Sua política foi desenhada para valorizar a cultura realizada na base da sociedade brasileira, a articulação em rede e a gestão compartilhada, com base nos princípios da autonomia, protagonismo e empoderamento da sociedade civil.</p>
            <p>A Política de Cultura Viva contempla iniciativas ligadas à cultura de base comunitária, Indígenas, Quilombolas, de Matriz Africana, economia solidária, produção cultural urbana e periférica, cultura digital, cultura  popular, com ampla incidência no segmento da juventude, abrangendo todos os tipos de linguagem artística e cultural como música, artes cênicas, cinema, circo, literatura, entre outras.</p>
            <p>Tem o objetivo de promover a articulação destas iniciativas em rede, contribuindo para a inclusão social, o combate ao preconceito e a todas as formas de discriminação e intolerância, o reconhecimento e a valorização da diversidade cultural brasileira e o pleno exercício dos direitos culturais.</p>
            <p>Desde 23 de julho de 2014, com a sanção da Lei Cultura Viva (Lei nº 13.018/2014), o Programa Cultura Viva e os Pontos e Pontões de Cultura tornaram-se Política de Estado. Além de garantir a continuidade do Programa, a Lei simplifica e desburocratiza os processos de prestação de contas e o repasse de recursos para as organizações da sociedade civil. Uma grande vitória para o movimento cultural brasileiro!</p>
        </div>
        <div class="row">
            <h3 id="ponto_e_pontao">O que é Ponto e Pontão de Cultura?</h3>
            <p>Pontos de Cultura são grupos, coletivos e entidades de natureza ou finalidade cultural que desenvolvem e articulam atividades culturais em suas comunidades, reconhecidos, certificados ou fomentados pelo Ministério da Cidadania por meio dos instrumentos da Política Nacional de Cultura Viva.
</p>
            <p>Pontões de Cultura são entidades de natureza ou finalidade cultural ou educativa que desenvolvem, acompanham e articulem atividades culturais articulando redes regionais, temáticas e identitárias de Pontos de Cultura e grupos culturais diversos. Ações de  mobilização,  troca de experiências e articulação entre os diferentes Pontos de Cultura que poderão se agrupar em nível estadual, regional ou por áreas temáticas, visando à capacitação, ao mapeamento e a ações conjuntas.</p>
            <p>Para se tornar Ponto ou Pontão de Cultura, basta aderir ao Cadastro Nacional de Pontos e Pontões de Cultura, seja por meio de acesso direto ao sítio eletrônico do Cadastro, ou se inscrevendo nos editais de apoio e fomento do Ministério da Cidadania e dos estados e municípios parceiros.</p>
        </div>
        <div class="row">
            <h3 id="por_que_ser_ponto_ou_pontao">Por que ser Ponto ou Pontão de Cultura?</h3>
            <p>O reconhecimento como Ponto ou Pontão de Cultura garante uma chancela institucional, que pode ser importante para a obtenção de apoios e parcerias, e permite que a entidade ou coletivo se articule com os outros pontos e pontões da rede, a partir de afinidades temáticas ou do pertencimento a um território. 
</p>
            <p>A Política Nacional de Cultura Viva conta ainda com uma série de modalidades de apoio e fomento direto às entidades e coletivos culturais. Os editais do Cultura Viva possibilitam o fomento direto a Pontos e Pontões de Cultura, prêmios de incentivo e reconhecimento às iniciativas culturais, bolsas para agentes culturais, entre outras modalidades de apoio financeiro direto.</p>
        </div>
        <div class="row">
            <h3 id="o_que_ganho">O que eu ganho com isso?</h3>
            <p>Ser um Ponto ou Pontão de Cultura é um reconhecimento, por parte do Estado, de uma ação cultural continuada já desenvolvida por uma entidade ou coletivo cultural. Na prática, essas iniciativas já são "Pontos de Cultura" e “Pontões de Cultura”, antes de serem certificadas pelo MinC, pelo diálogo que estabelecem com sua comunidade e sua capacidade de atuação social e transformação do território.</p>
            <p>A Rede Cultura Viva é um ambiente de trocas, colaborações, articulações e encontros. Os Pontos e Pontões se fortalecem quando se articulam em rede e compartilham espaços, metodologias, produtos, serviços, processos criativos e ações coletivas.</p>
            <p>Os Pontos e Pontões de Cultura, articulados em rede, são co-gestores das políticas culturais e, nesse sentido, os espaços de participação social são fundamentais: encontros, fóruns, Teias, são instâncias de gestão compartilhada, onde as redes estão sempre em movimento. </p>
            <p>Ao preencher este Cadastro, você poderá formar parte da Rede Cultura Viva! A partir dele você estará se conectando e conhecendo mais pessoas e iniciativas que possuem a mesma vontade de produzir, se articular e de criar alternativas de vida e de trabalho empreendedor no campo da cultura.</p>
            <p>Ao se cadastrar na Rede Cultura Viva você e seu Ponto/Pontão de Cultura terão acesso a informações, processos e experiências, que permitirão o compartilhamento de capacidades, conhecimentos, serviços e produtos conectados em rede. 
</p>
        </div>
        <div class="row">
            <h3 id="criterios">Quais são os Critérios para a autodeclaração?</h3>
            <p>A autodeclaração de entidades e coletivos culturais como Pontos ou Pontões de Cultura é uma iniciativa da sociedade civil, e cabe ao Ministério da Cidadania, estados e municípios oferecerem estímulo, apoio e instrumentos para que, a partir desta iniciativa de autodeclaração, as entidades e coletivos possam ser certificados e reconhecidos como Pontos ou Pontões  de Cultura, conforme previsto na Lei Cultura Viva e em sua regulamentação. </p>
            <p>A plataforma Rede Cultura Viva é a interface digital do Cadastro Nacional dos Pontos e Pontões de Cultura, instrumento previsto na Lei Cultura Viva, conforme disposto no Inciso III do artigo 4 desta lei:</p>
            <p>"III - Cadastro Nacional de Pontos e Pontões de Cultura: integrado pelos grupos, coletivos e pessoas jurídicas de direito privado sem fins lucrativos que desenvolvam ações culturais e que possuam certificação simplificada concedida pelo Ministério da Cidadania. "</p>
            <p>A Certificação Simplificada como Ponto ou Pontão de Cultura, através da Plataforma Rede Cultura Viva, será um processo seletivo simplificado, aberto permanentemente em fluxo contínuo, destinado ao reconhecimento de entidades e coletivos culturais como Pontos ou Pontões de Cultura. As propostas de certificação enviadas serão analisadas por uma comissão paritária composta por membros do poder público e da sociedade civil. A análise será realizada com base nos seguintes critérios, alinhados às diretrizes e normativos da Politica Nacional de Cultura Viva:
</p>
            <p><strong>Critérios:</strong></p>
            <p>I – Ser pessoa jurídica de direito privado sem fins lucrativos, ou coletivo cultural sem constituição jurídica, de natureza ou finalidade cultural, que desenvolva e articule atividades culturais em suas comunidades;</p>
            <p>II  - Comprove, no mínimo, 1 (um) ano de existência e desenvolvimento de atividade culturais, através de fotos, material gráfico de eventos, publicações impressas e em meios eletrônicos e outros materiais comprobatórios; </p>
            <p>III  - Que tenha o apoio, por meio de cartas de reconhecimento, de no minimo 2 (dois) Pontos de Cultura, instituições públicas, privadas, coletivos culturais ou comunidades, com ou sem constituição jurídica, relacionados à arte, cultura, educação ou desenvolvimento comunitário. As cartas devem ser assinadas e digitalizadas. Serão aceitas somente assinaturas manuscritas em papel ou impressões digitais em caso de pessoas não alfabetizadas. Não serão aceitas assinaturas digitais. 
IV - Preencha as informações de Cadastro na Plataforma Rede Cultura Viva e aceite os Termos de Uso e o Termo de Adesão à Política Nacional de Cultura Viva.</p>
            <p>Para chegarmos aos 15 mil Pontos de Cultura fomentados em 2020, conforme previsto nas metas do Plano Nacional de Cultura, é preciso ampla mobilização da sociedade, e cabe ao Ministério da Cidadania promover condições para mapear, reconhecer, dar visibilidade e promover intercâmbios e trocas em rede entre estas iniciativas culturais de todo o país. </p>
            <p>A aprovação da Lei Cultura Viva foi uma conquista da sociedade,  e os pontos de cultura são uma experiência de política cultural reconhecida nacional e internacionalmente. O reconhecimento de entidades e coletivos culturais como pontos de cultura através da auto-declaração será fundamental para ampliar a escala, dar visibilidade e democratizar ainda mais esta experiência, alcançando os mais diversos segmentos da diversidade cultural brasileira. 
</p>
        </div>
        <div class="row">
            <h2>Curtiu? Então seja mais um ponto nesta teia da<br/>diversidade cultural brasileira!</h2>
            <form method="POST" action="<?php echo $app->createUrl('cadastro', 'registra') ?>">
                <button type="submit" class="btn btn__active">Entrar na Rede</button>
            </form>
        </div>
    </article>
    <a href="#" class="btn-volta-topo"><img src="<?php $this->asset('img/seta_topo.png') ?>" /></a>
</section>
